apps/server/src/modules/mail/mail.service.ts

```diff
@@ -1,19 +1,21 @@
-import { Injectable } from '@nestjs/common';
+import { Injectable, Inject } from '@nestjs/common';
 import { AmqpConnection } from '@golevelup/nestjs-rabbitmq';
 
-import { Configuration } from '@hpi-schul-cloud/commons';
+import { Mail } from './mail.interface';
 
-import { Mail } from './interfaces/mail';
+interface MailServiceOptions {
```

Comment:
extract [all] interfaces for reusability without circular dependencies into interfaces folder of the module and import them afterwards